index.Rmd

```diff
@@ -69,7 +69,7 @@ We've made a few Sheets available to "anyone with a link", for use in examples a
 ```{r}
 sheets_examples()
 
-sheets_examples("gap")
```

Comment:
I think I want the plural here. I'm trying to show that it's OK for the regex to match more than 1 name.